packages/kbn-mapbox-gl/src/index.ts

```diff
@@ -25,13 +24,15 @@ import type {
   Point,
   CustomLayerInterface,
 } from 'maplibre-gl';
-import mapboxgl from 'maplibre-gl/dist/maplibre-gl-csp';
+// @ts-expect-error
+import mapboxglDist from 'maplibre-gl/dist/maplibre-gl-csp';
 // @ts-expect-error
 import mbRtlPlugin from '!!file-loader!@mapbox/mapbox-gl-rtl-text/mapbox-gl-rtl-text.min.js';
 // @ts-expect-error
 import mbWorkerUrl from '!!file-loader!maplibre-gl/dist/maplibre-gl-csp-worker';
 import 'maplibre-gl/dist/maplibre-gl.css';
 
+const mapboxgl: any = mapboxglDist;
```

Comment:
curious: Does this `any` impact anything downstream wrt type-checking?

Reply:
The api extractor currently does not support the export of things like `declare module` so in this case and while they dont have that feature, this is needed :)